interoperator/pkg/constants/constants.go

```diff
@@ -13,9 +13,10 @@ const (
 	LastOperationKey                      = "interoperator.servicefabrik.io/lastoperation"
 	ErrorThreshold                        = 10
 
-	ConfigMapName   = "interoperator-config"
-	ConfigMapKey    = "config"
-	ProvisionerName = "provisioner"
+	ConfigMapName         = "interoperator-config"
+	ConfigMapKey          = "config"
+	ProvisionerName       = "provisioner"
+	LeaderProvisionerName = "provisioner-template"
```

Comment:
Now a better name for the constant would be `ProvisionerTemplateName`

Reply:
Yes. `ProvisionerTemplateName` sounds better.
Updated :white_check_mark: